Dockerfile

```diff
@@ -72,7 +72,8 @@ COPY holohub /tmp/scripts/
 RUN mkdir -p /tmp/scripts/utilities
 COPY utilities /tmp/scripts/utilities/
 RUN chmod +x /tmp/scripts/holohub
-RUN /tmp/scripts/holohub setup && rm -rf /var/lib/apt/lists/*
+RUN ln -sf /tmp/scripts/holohub /usr/local/bin/holohub
+RUN holohub setup && rm -rf /var/lib/apt/lists/*
```

Comment:
why?

Reply:
Side effect of `lint.sh` investigation, thought it would be nice to have but is not critical to these changes. I'll move to a separate branch to avoid confusion.

Reply:
Scrapping the minor change on this line altogether, looks like we have HoloHub CLI both `COPY`-ed under `tmp/scripts` and also mounted with `./holohub run-container`, not ideal but probably OK to leave for now

Reply:
Removed 👍